README.md

```diff
@@ -271,6 +271,12 @@ $ nix-env --install ripgrep
 $ # (Or using the attribute name, which is also ripgrep.)
 ```
 
+If you're a **Guix** user, you can install ripgrep from the official [package collection](https://guix.gnu.org/en/packages/ripgrep-13.0.0/):
```

Comment:
Please follow the style of the surrounding code and wrap to 79 columns (inclusive). If the link is too long, that's okay, but it should go on its own line. (Just follow the surrounding style.)

Reply:
Hi, I pushed a update. Thanks for the review!